js/map_utils.js

```diff
@@ -185,10 +186,22 @@ function documentCategorySwitch(category, prevCategory, action) {
 }
 
 
+function highlightCircle(currentCircle) {
```

Comment:
I don't know what this is...

Reply:
it's what makes the circle go from slightly see-through to solid when you hover

Reply:
not sure how it is being added here though...they should've been there the whole time. Same with `unhighlightCircle` below

Reply:
but how did it get in my pull request? probably something I did on the merge conflict

Reply:
yes, i think this is a merge thing, probably because i moved highlightCircle and unhighlightCircle to circles.js